src/components/Markdown/MarkdownLink.tsx

```diff
@@ -37,8 +43,23 @@ const MarkdownLink: React.FC<{
     return <YouTubeEmbed embedUrl={href} />;
   }
 
-  // HACK to track clicks on the exposure notification recommendation
+  if (isInternalEmbed(href)) {
+    return (
+      <CenterEmbed>
+        <iframe
+          src={href}
+          title="Covid Act Now"
+          width="400"
+          height="440"
```

Comment:
these are the dimensions of the homepage embed. do we need to have different iframes/dimensions depending on homepage vs location page embed?

Reply:
actually, i guess just using the larger of the sets of dimensions should work for both